Test failure in test-long etc. because of singular

### Steps To Reproduce

Log on CI https://github.com/sagemath/sage/actions/runs/15081532937/job/42398821050?pr=39973 :

```
2025-05-17T04:17:53.5248491Z #8 184.2 Trying to install singular
2025-05-17T04:17:54.5097355Z #8 185.2 Trying to install singular-doc
2025-05-17T04:17:55.4740373Z #8 186.2 Trying to install libsingular4-dev

2025-05-17T04:19:24.8961848Z #25 12.32 -----------------------------------------------------------------------------
2025-05-17T04:19:24.8962573Z #25 12.32 Checking whether SageMath should install SPKG singular...
2025-05-17T04:19:24.8963289Z #25 12.32 real_configure: singular has already been installed by SageMath
2025-05-17T04:19:24.8964097Z #25 12.32 -----------------------------------------------------------------------------

2025-05-17T04:19:29.7954224Z #25 17.33 singular:                       installed SPKG version 4.4.0 will be replaced by version 4.4.1
2025-05-17T04:19:29.7954860Z #25 17.34 singular_jupyter:               no; optional, use "./configure --enable-singular_jupyter" to install SPKG version 0.9.7

2025-05-17T05:31:46.2297981Z #31 4321.4 make --no-print-directory singular-SAGE_LOCAL-no-deps
2025-05-17T05:31:46.2298625Z #31 4321.4 [singular-4.4.1] installing. Log file: /sage/logs/pkgs/singular-4.4.1.log
2025-05-17T05:31:46.2299399Z #31 4321.4   [singular-4.4.1] successfully installed (real 16m13.156s user 8m33.854s sys 1m25.894s).

2025-05-17T05:31:46.3195967Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] mkdir -p en/reference/repl
2025-05-17T05:31:46.3196201Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] sage -advanced > en/reference/repl/options.txt
2025-05-17T05:31:46.3196428Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] make doc-inventory--reference-references
2025-05-17T05:31:46.3196704Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] sage --docbuild --no-pdf-links reference/references inventory 
2025-05-17T05:31:46.3196898Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] Error building the documentation.
2025-05-17T05:31:46.3197088Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] Traceback (most recent call last):
2025-05-17T05:31:46.3197337Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "<frozen runpy>", line 198, in _run_module_as_main
2025-05-17T05:31:46.3197557Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "<frozen runpy>", line 88, in _run_code
2025-05-17T05:31:46.3197832Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage_docbuild/__main__.py", line 552, in <module>
2025-05-17T05:31:46.3197999Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     sys.exit(main())
2025-05-17T05:31:46.3198249Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]              ^^^^^^
2025-05-17T05:31:46.3198518Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage_docbuild/__main__.py", line 545, in main
2025-05-17T05:31:46.3198873Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     import sage.all  # TODO: Remove once all modules can be imported independently  # noqa: F401
2025-05-17T05:31:46.3199031Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     ^^^^^^^^^^^^^^^
2025-05-17T05:31:46.3199268Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage/all.py", line 96, in <module>
2025-05-17T05:31:46.3199464Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     from sage.schemes.all import *
2025-05-17T05:31:46.3199716Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage/schemes/all.py", line 25, in <module>
2025-05-17T05:31:46.3199964Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     from sage.schemes.hyperelliptic_curves.all import *
2025-05-17T05:31:46.3200299Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage/schemes/hyperelliptic_curves/all.py", line 1, in <module>
2025-05-17T05:31:46.3200628Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     from sage.schemes.hyperelliptic_curves.constructor import HyperellipticCurve
2025-05-17T05:31:46.3200992Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage/schemes/hyperelliptic_curves/constructor.py", line 17, in <module>
2025-05-17T05:31:46.3201275Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     from .hyperelliptic_generic import HyperellipticCurve_generic
2025-05-17T05:31:46.3201672Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage/schemes/hyperelliptic_curves/hyperelliptic_generic.py", line 36, in <module>
2025-05-17T05:31:46.3201945Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     import sage.schemes.curves.projective_curve as plane_curve
2025-05-17T05:31:46.3202280Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "/sage/src/sage/schemes/curves/projective_curve.py", line 144, in <module>
2025-05-17T05:31:46.3202704Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     from sage.libs.singular.function import singular_function, lib as singular_lib, get_printlevel, set_printlevel
2025-05-17T05:31:46.3203057Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]   File "sage/libs/singular/function.pyx", line 98, in init sage.libs.singular.function
2025-05-17T05:31:46.3203369Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install]     from sage.libs.singular.option import opt_ctx
2025-05-17T05:31:46.3203949Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] ImportError: libSingular-4.4.0.so: cannot open shared object file: No such file or directory
2025-05-17T05:31:46.3204245Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] make[6]: *** [Makefile:28: doc-inventory--reference-references] Error 1
2025-05-17T05:31:46.3204500Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] make[5]: *** [Makefile:44: doc-inventory-reference] Error 2
2025-05-17T05:31:46.3204750Z #31 4321.4   [sagemath_doc_html-none]   [spkg-install] make[5]: Target 'doc-html' not remade because of errors.
```

Also mentioned in https://github.com/sagemath/sage/pull/40085 and https://github.com/sagemath/sage/pull/39973#issuecomment-2888486035

--- a/.github/workflows/docker.yml
+++ b/.github/workflows/docker.yml
@@ -20,6 +20,7 @@ on:
           [
            "ubuntu-focal",
            "ubuntu-jammy",
+           "ubuntu-noble",
            "debian-bullseye",
            "debian-bookworm",
            "fedora-40",

